module/Decision/src/Decision/Model/Member.php

```diff
@@ -60,11 +60,30 @@ class Member
     /**
      * Organs this member is in.
      *
-     * @ORM\ManyToMany(targetEntity="Decision\Model\Organ", mappedBy="members")
+     * @ORM\ManyToMany(targetEntity="Decision\Model\Organ", mappedBy="members", fetch="EAGER")
      */
     protected $organs;
 
     // TODO: Much, MUCH more fields
+	
+	/**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->organs = new ArrayCollection();
```

Comment:
Deze regel geeft een error in travis. Namelijk, de `ArrayCollection` kan hij hier niet vinden. Fix dit met een `use` regel.

Also, zo veel aanpassen aan de `Decision` module heeft geen zin. Deze ondergaat binnen korte tijd een grote rewrite, als de database klaar is. Wat er tot nu toe in gestaan heeft, probeer ik mee compatibel te blijven. Maar jouw wijzigingen neem ik daar niet in mee.